semeio/workflows/localisation/local_script_lib.py

```diff
@@ -744,6 +777,155 @@ def add_ministeps(
         ert_local_config.getUpdatestep().attachMinistep(ministep)
 
 
+def get_corr_group_spec(correlations_spec_list, name):
+    found = False
+    corr_spec = None
```

Comment:
Dont need to set this to `None`, if it is not `found` this is not used. The exception is if you would like to get rid of `found` and just to a check for `if not corr_spec` instead. That might be better, assuming there can not be `None` in `correlations_spec_list`

Reply:
I keep the found variable and remove the corr_spec = None assignment.